src/core/friendly_errors/sketch_reader.js

```diff
@@ -319,12 +319,14 @@ if (typeof IS_MINIFIED !== 'undefined') {
       }
     }
     const keyArray = Object.keys(p5Constructors);
+    const globalFunctions = ['Renderer', 'Renderer2D', 'RendererGL'];
```

Comment:
more classes can be added as per need

Reply:
right, maybe something like `classesWithGlobalFunctions` would be more accurate?